patches/0070-Temporarily-undo-aws.iam.Role-managed_policy_arns-an.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Anton Tayanovskyy <anton@pulumi.com>
+Date: Fri, 25 Oct 2024 13:17:15 -0400
+Subject: [PATCH] Temporarily undo aws.iam.Role managed_policy_arns and
+ inline_policy deprecations
```

Comment:
Is it intentional that we're removing the `inline_policy` deprecation as well? This was introduced a couple of releases ago: https://github.com/pulumi/pulumi-aws/pull/4571

Reply:
You are right. It's currently published. My mistake, my objective here is to minimize changes in this PR before we have a chance to have an informed conversation. I'll let inline policy stay deprecated, one sec.

Reply:
Awesome!